packages/ckeditor5-theme-lark/theme/ckeditor5-ui/globals/_colors.css

```diff
@@ -16,7 +16,8 @@
 
 	/* -- Generic colors ------------------------------------------------------------------------ */
 
-	--ck-color-focus-border: 									hsl(208, 79%, 51%);
+	--ck-color-focus-border-coordinates: 						208, 79%, 51%;
```

Comment:
This should be noted as minor breaking change for integrations that override `--ck-color-focus-border`